profiles/coreos/base/package.use

```diff
@@ -39,7 +39,7 @@ net-analyzer/nmap ncat -lua
 app-admin/sudo -sendmail
 
 # avoid pulling in gnutls, disable gentoo-only bits, enable journal upload
-sys-apps/systemd -ssl curl vanilla lzma
+sys-apps/systemd -ssl curl vanilla -lz4 lzma gcrypt
```

Comment:
On Feb 21, 2015 8:38 AM, "Alex Crawford" <notifications@github.com> wrote:
>
> In profiles/coreos/base/package.use:
>
> > @@ -39,7 +39,7 @@ net-analyzer/nmap ncat -lua
> >  app-admin/sudo -sendmail
> >
> >  # avoid pulling in gnutls, disable gentoo-only bits, enable journal
upload
> > -sys-apps/systemd -ssl curl vanilla lzma
> > +sys-apps/systemd -ssl curl vanilla -lz4 lzma gcrypt
>
> I was hoping to make few changes to the upstream ebuild. Gentoo adopted
lz4, while everyone else went with lzma.

Is it reasonable to support both or are you avoiding a new dependency?